keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactVeto.kt

```diff
@@ -11,7 +11,15 @@ data class EnvironmentArtifactVeto(
   val version: String,
   val vetoedBy: String?,
   val comment: String?
-)
+) {
+  init {
+    if (comment != null) {
```

Comment:
Instead of repeating this code in 3 different places, how about declaring a `validateComment` function in a shared `validations.kt` module that you can call from all places?

Reply:
honestly, I think it's more complicated and less readable, for this simple use case, as it's a simple data class with no dependencies on other classes (validators), but it's only my opinion..